tests/fsharpqa/Source/Conformance/LexicalAnalysis/StringsAndCharacters/UnicodeString01.fs

```diff
@@ -1,11 +1,29 @@
-// #Conformance #LexicalAnalysis 
-#light
-
 // Test string literals with short Unicode Literals
 
-let unicodeString = "\u2660 \u2663 \u2665 \u2666"
-let expectedResult = "♠ ♣ ♥ ♦"
+let mutable failure = false
+
+let checkStr (inputStr:string) expectedChars expectedStr =
```

Comment:
could you move this tests in FSharp.Compiler.Unittests? this would make it easy to run the test from testrunners like nCrunch.